lib/tessel/deployment/rust.js

```diff
@@ -60,5 +60,6 @@ exportables.preRun = function(tessel) {
   });
 };
 
+// exportables.postRun = function(tessel, remote) {};
```

Comment:
This is here as "pseudo documentation" until the language support API churn slows and we can start documenting properly